lucetc/src/load.rs

```diff
@@ -18,7 +19,11 @@ pub fn read_module(
                 .ok_or(Error::Signature("public key is missing".to_string()))?,
         )?;
     }
-    read_bytes(contents)
+    if translate_wat {
+        read_bytes(contents)
+    } else {
+        Ok(contents)
```

Comment:
we ought to still check `wasm_preamble` in the no-`wat` case, since the only thing Lucet should accept is a `wasm`.

Reply:
Fixed. Thanks